packages/account/src/Sections/Profile/PhoneVerification/otp-verification.tsx

```diff
@@ -38,19 +43,32 @@ const ConfirmYourEmail = observer(() => {
     return (
         <PhoneVerificationCard is_small_card>
             <Text bold>
-                <Localize i18n_default_text="Confirm it's you" />
+                {is_email_verified ? (
+                    <Localize i18n_default_text='Verify your number' />
+                ) : (
+                    <Localize i18n_default_text="Confirm it's you" />
+                )}
             </Text>
             <div className='phone-verification__card--email-verification-content'>
                 <Text size='sm'>
-                    <Localize
-                        i18n_default_text="We've sent a verification code to <0>{{users_email}}</0>."
-                        values={{ users_email: email }}
-                        components={[<strong key={0} />]}
-                    />
-                </Text>
-                <Text size='sm'>
-                    <Localize i18n_default_text='Enter the code or click the link in the email to verify that the account belongs to you.' />
+                    {is_email_verified ? (
+                        <Localize
+                            i18n_default_text='Enter the 6-digit code sent to you via {{phone_verification_type}} at {{users_phone_number}}:'
+                            values={{ phone_verification_type, users_phone_number: phone }}
+                        />
+                    ) : (
+                        <Localize
+                            i18n_default_text="We've sent a verification code to <0>{{users_email}}</0>."
+                            values={{ users_email: email }}
+                            components={[<strong key={0} />]}
+                        />
+                    )}
                 </Text>
+                {!is_email_verified && (
```

Comment:
By any chance can you move this in else condition after line number `64`.  👀 
By doing this you can get rid of adding one more condition. 😬

Reply:
Good catch ! will edit it 👌

Reply:
Thank you @suisin-deriv 🙇